docs/compute-engine/quickstart_android.md

```diff
@@ -1,15 +1,12 @@
 # Larq Compute Engine Android Quickstart
 
-This guide consists of two main sections. In the [first section](#create-your-own-android-app-using-lce-and-tensorflow-lite),
-we describe how to build your own Android app using Larq Compute Engine (LCE) and
+This guide describes how to build your own Android app using Larq Compute Engine (LCE) and
 [TensorFlow Lite Java Inference APIs](https://www.tensorflow.org/lite/guide/inference#load_and_run_a_model_in_java)
 to perform inference with a model built and trained with [Larq](https://larq.dev).
-This can be achieved either by using our pre-built [LCE Lite AAR hosted on Bintray](https://bintray.com/plumeraihq/larq-compute-engine)
-(see [here](#use-the-lce-lite-aar-from-bintray) for instructions) or you can build the LCE Lite AAR on
-your local machine by following the guide [here](#build-lce-lite-aar-locally).
-In the [second section](#build-an-lce-inference-binary), we describe how to build
-a LCE-compatible inference binary that can be executed on Android OS
-(e.g, a binary to benchmark your models).
+This can be achieved either by using our pre-built [LCE Lite AAR hosted on Bintray](https://bintray.com/plumeraihq/larq-compute-engine), or you can build the LCE Lite AAR on your local machine (see [here](#2-add-lce-compatible-tensorflow-lite-aar-to-the-project) for instructions for either approach).
+
+If you'd rather build a binary than a full Android app, [this guide](/compute-engine/build_android) describes how to build
+a LCE-compatible inference binary that can be executed on Android OS (e.g to [benchmark your models](/compute-engine/benchmark)).
```

Comment:
If I recall correctly, if we omit this trailing `/` then the link works but links on *that* page then don't work anymore.
```suggestion
If you'd rather build a binary than a full Android app, [this guide](/compute-engine/build_android/) describes how to build
a LCE-compatible inference binary that can be executed on Android OS (e.g to [benchmark your models](/compute-engine/benchmark/)).
```

Reply:
Interesting, it works locally (it automatically adds the slash even if it isn't in the link)

Reply:
We ran into problems with mkdocs before, but we've fixed it on the server in #88

Reply:
Apparently this works since https://github.com/larq/docs/pull/88/files, slashless pages are automatically redirected